app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,7 @@
   <div class="nav_bar">
     <%= link_to "Sign out", destroy_user_session_path %>
     <%= link_to "Home", root_path%>
-    <%= link_to "Posts", posts_path %>
+    <%= link_to "All Posts", posts_path %>
```

Comment:
Updated nav bar to differentiate between this page that shows all posts and the Hello page that shows 5.